src/components/dropDownStops.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Button, Text } from 'react-native-paper'
+import DropDown from 'react-native-paper-dropdown'
+import { Station } from '../screens/home'
+
+type StopsDropDownProps = {
+  setResult: Dispatch<SetStateAction<string>>
+  handleResults: (st: Station) => void
+}
+const listOfStops: { label: string; value: string }[] = []
```

Comment:
I'm not a fan of mutating global variables like this. I think you should just have `stopNames` as a global variable, and have a function that generates and returns a "list of stops" which you call in the component.